backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -457,6 +473,17 @@ public void deleteOrganization(Organization org) {
     return getOrganizationRoleClaimsForUser(user);
   }
 
+  private Optional<OrganizationRoleClaims> getOrganizationRoleClaimsFromTenantDataAccess(
```

Comment:
this method feels misnamed -- it's really just getting org role claims from the authorities, right? those authorities just happen to be derived from tenant data access in most of the places this function is called